plugins/pathable.cpp

```diff
@@ -248,12 +332,13 @@ static void check_wagon_tile(FloodCtx & ctx, const df::coord & pos) {
     ctx.seen.emplace(pos);
 
     if (ctx.entry_tiles.contains(pos)) {
-        ctx.wagon_path.emplace(pos);
+        ctx.wagon_path.emplace(pos); // Is this needed?
```

Comment:
probably not since the rendering of the edge tile will overwrite the rendering of the wagon path

Reply:
actually, keep this in, since we might want to highlight the overlap (by coloring it green?)